single-function/add-two-numbers/02-impl-revision.js

```diff
@@ -0,0 +1,24 @@
+const addTwoNumbers = (l1, l2) => {
+  let carryover = 0,
+    dummyhead = { val: 0, next: null },
+    curr = dummyhead;
+
+  while (l1 !== null || l2 !== null) {
+    let list1Value = l1 !== null ? l1.val : 0;
```

Comment:
can these be `const` -- we should favor `const` wherever possible.

Reply:
all possible variables set to const, only carryOver and curr needed to remain let